SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -46,9 +81,40 @@ void SpatialVirtualWorkerTranslator::OnComponentUpdated(const Worker_ComponentUp
 	}
 }
 
-// The translation schema is a list of Mappings, where each entry has a virtual and physical worker ID. 
+void SpatialVirtualWorkerTranslator::AuthorityChanged(const Worker_AuthorityChangeOp& AuthOp)
+{
+	const bool bAuthoritative = AuthOp.authority == WORKER_AUTHORITY_AUTHORITATIVE;
```

Comment:
Do we need to cache bAuthoritative? If so could we move it inside the proceeding if statement.

Reply:
I think we'll use it again in the ACL block, so this would be the best scope for it. If you don't end up using it in the Enforcer, you can remove it.